### DISCUSSION TITLE
Thank you for the improved releases (and some feedback/ideas)

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#23836] [New to GitHub]
<p>Just want to drop my thanks towards the team that made the release improvements.<br>
The implementation of <a href="https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes" rel="noopener nofollow ugc">automatic Release drafts</a> based on labels is a real nice thing. Already loved the release-drafter action and I can see this new feature become a nice addition too.</p>
<p>The only things I hope for now are the following:</p>
<ul>
<li>Option to set a general layout. By that I mean to have a way to define a structure for the release like a pre-made title, general body and where the categories end up.</li>
<li>Option to automatically set a (sem)version based on applied labels.</li>
</ul>
<p>Basically, similar features to what the release-drafter action currently offers. This would make the process of creating releases a lot more streamlined and easy to do!<br>
But other than that do I like where this is going, so keep up the good work (And if you can, give the responsible devs some hugs from me. Thanks)!</p>

**Comment by MylesBorins** [comment, marked as answer]
<p>We explore making the general layout more configurable as an option but supporting this feature would require introducing variables into the configuration which was something that was out of scope for this iteration. The good news is that we have built the infrastructure in such a way that we could support this in the future!</p>
<p>Regarding selecting the tag automatically… right now we don’t do anything with regards to creating release commits and in many cases for distributed software there are files (like package.json) that need to be updated with the new tag as part of the release. There are also a large number of versioning schemes folks use including semver, pep440, and calver… and the possibility of doing the wrong thing is quite high. As releases are one of the most important part of the software development process we didn’t want to invest in a feature that had a high probability of having lots of incorrect results.</p>
